src/system/tests/TestTLVPacketBufferBackingStore.cpp

```diff
@@ -35,6 +35,18 @@ using namespace ::chip;
 
 namespace {
 
+void WriteUntilRemainingLessThan(nlTestSuite * inSuite, PacketBufferTLVWriter & writer, const uint32_t & remainingSize)
+{
+    CHIP_ERROR error;
```

Comment:
nit: move this where you need this instead of top-level?

```cpp
NL_TEST_ASSERT(
    inSuite,
    writer.Put(TLV::AnonymousTag(), static_cast<uint8_t>(7)) == CHIP_NO_ERROR);
```